samples/performance/multithreading_render_passes/multithreading_render_passes.cpp

```diff
@@ -52,7 +52,7 @@ bool MultithreadingRenderPasses::prepare(vkb::Platform &platform)
 		shadow_render_targets[i] = create_shadow_render_target(SHADOWMAP_RESOLUTION);
 	}
 
-	load_scene("scenes/bonza/Bonza4X.gltf");
+	load_scene("scenes/sponza/Sponza01.gltf");

```

Comment:
Did you intend to change the scene here?
This scene is a lot lighter and I don't really see any performance difference between the modes now.

Reply:
No, sorry. It's fixed now.